doc/crud.md

```diff
@@ -268,6 +365,11 @@ This is the simplest case as we don't need to worry about previous revisions.
     CDTDocumentRevision *saved = [datastore createDocumentFromRevision:rev
                                                                  error:&error];
     ```
+    ```swift
+    let rev = CDTDocumentRevision("doc1")
+    rev.body = [....]
+    let saved = try datastore.createDocumentFromRevision(rev)
+    ```
 
 1. Add a new document to the store with attachments.
```

Comment:
For some reason the rendering is broken by this change https://github.com/cloudant/CDTDatastore/blob/swift2-crud/doc/crud.md#creating-a-new-document.